Private/Move-ArchivedLogs.ps1

```diff
@@ -6,14 +6,14 @@ function Move-ArchivedLogs {
         [string] $DestinationPath
     )
     $NewSourcePath = "\\$ServerName\$($SourcePath.Replace(':\','$\'))"
-    $PathExists = Test-Path $NewSourcePath
-    if ($PathExists) {
-        Write-Color @script:WriteParameters '[i] Moving log file from ', $NewSourcePath, ' to ', $DestinationPath -Color White, Yellow, White, Yellow
-        Move-Item -Path $NewSourcePath -Destination $DestinationPath
-        if (!$?) {
-            Write-Color @script:WriteParameters '[i] File ', $NewSourcePath, ' couldn not be moved.' -Color White, Yellow, White
+    if (Test-Path $NewSourcePath) {
```

Comment:
Just a thing I've noticed lately if $NewSourcePath would be an empty string it would throw an error on Test-Path. Maybe different wrapper in PSSharedGoods for Test-Path for that? Of course, in this case, it won't happen but in my other tools, I've often checked Test-Path and then find out it would error out.